openedx/features/course_duration_limits/config.py

```diff
@@ -1,14 +1,7 @@
 """
 Content type gating waffle flag
 """
-import random
-
-from django.dispatch import receiver
-from django.db import IntegrityError
-
-from experiments.models import ExperimentData, ExperimentKeyValue
-from student.models import EnrollStatusChange
-from student.signals import ENROLL_STATUS_CHANGE
+from __future__ import absolute_import
 
```

Comment:
^ Since we are not importing anything, I think we can remove the absolute_import as well.

Reply:
Yes, I didn't notice that. Thanks!

Reply:
It seems after I made these changes, the tests are broken. I add it back for testing and will check the cause behind it.